app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -394,8 +401,26 @@ object AppRepository {
     fun readAlarmTimeIndex() =
             sharedPreferencesRepository.getAlarmTimeIndex()
 
-    fun readAlarmToneUri() =
-            sharedPreferencesRepository.getAlarmTone().toUri()
+    /**
+     * Returns the persisted alarm tone, the default alarm tone URI or `null` aka. "silent".
```

Comment:
```suggestion
     * Returns the alarm tone `Uri` or `null` for silent alarms.
```

I find the original text a little confusing and too close to the implementation details.

Reply:
:heavy_check_mark: Redacted in 23b10ca4822a133ed18915842987e132b4a9205a.